daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -224,9 +232,15 @@ object Node {
       signatories: Set[Party],
       stakeholders: Set[Party],
       key: Option[KeyWithMaintainers[Val]],
-      override val byKey: Boolean // invariant (!byKey || exerciseResult.isDefined)
+      override val byKey: Boolean, // invariant (!byKey || exerciseResult.isDefined)
+      // keep version the last field of the class
```

Comment:
Why?

Reply:
I would like to have a consistent position over all the node. Since ledger implementation should not care too much of this field I though it is better at the end.

Reply:
I see. Maybe it would help to make the comment more explicit along the lines of
```
// For the sake of consistency between types with a version field, keep this field the last.
```

Reply:
Changed.